src/outputs/outputs_main.py

```diff
@@ -152,10 +154,11 @@ def run_outputs(  # noqa: C901
     # Running Intram by ITL (GB)
     if config["global"]["output_intram_gb_itl"]:
         OutputMainLogger.info("Starting Intram by ITL (GB) output...")
-        output_intram_by_itl(
+        intram_tot_dict = output_intram_by_itl(
             outputs_df,
             ni_full_responses,
             config,
+            intram_tot_dict,
             write_csv,
             run_id,
         )
```

Comment:
As above - intram_tot_dict missing line 180

Reply:
again, not sure about including GB. We could ask Dan and add it later perhaps?

Reply:
silly me! Good thing you noticed!